<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tstClass</class>
 <widget class="QMainWindow" name="tstClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tst</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Search:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>&amp;Add</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnDelete">
      <property name="text">
       <string>&amp;Delete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLayerWidget" name="layerWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLayerWidget</class>
   <extends>QListWidget</extends>
   <header>QLayerWidget.h</header>
   <slots>
    <slot>filter(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tst.qrc"/>
 </resources>
 <connections/>
</ui>
